IOMC/RandomEngine/src/TRandomAdaptor.h

```diff
@@ -23,45 +23,45 @@ namespace edm {
     virtual ~TRandomAdaptor();
 
     // Returns a pseudo random number in ]0,1[ (i. e., excluding the end points).
-    double flat() { return trand_->Rndm(); }
+    double flat() override { return trand_->Rndm(); }
```

Comment:
@Dr15Jones While the addition of the missing "override" qualifiers are not forced by the value_ptr change, they are connected. On the first attempt at this PR, I had removed the const qualifuer from the "put" member function, and the tests failed. I tracked the problem down to the fact that the "put" function no longer overrode the "put" function in the base class. So, the missing override caused me to make an error. It seemed silly to just add the override to "put" without also adding it to the other member functions that needed it.
I agree that the overrides could have been added in a separate commit, however.